lib/cisco_node_utils/command_reference_common.yaml

```diff
@@ -633,6 +633,19 @@ radius_server:
     config_set: '<state> radius-server host <ip> key <key>'
     default_value: null
 
+radius_server_group:
+  groups:
+    config_get: "show running-config radius all"
+    config_get_token: '/^aaa group server radius (.*) $/'
+    config_set: '<state> aaa group server radius <name>'
+    default_value: ''
+
+  hosts:
```

Comment:
Should you all this servers instead?  The cli and your methods refer to them as servers.